add-on/chrome/modules/MozLoopService.jsm

```diff
@@ -1348,17 +1348,26 @@ this.MozLoopService = {
     LoopRooms.on("update", onRoomsChange);
     LoopRooms.on("delete", onRoomsChange);
     LoopRooms.on("joined", (e, room, participant) => {
+      // The participant that joined isn't necessarily included in room.participants (depending on
+      // when the broadcast happens) so concatenate.
+      let isOwnerInRoom = room.participants.concat(participant).some(p => p.owner);
```

Comment:
I'm concerned about possible concatenation side-effects without cloning if room is a reference.  

Please separate this out into its own getter, with a big XXX describing why we're not using the existing roomsParticipantsCount.